### ISSUE TITLE
cache

### ISSUE THREAD

**Issue by dcuneo** [spantaleev/ckeditor-imagebrowser#10]
I like the plugin, but every time I add a new image to the directory (through some other means) and them call this plugin, it doesn't find the new image.  I then delete (IE tools) the temporary Internet files, and then call the this plugin, it finds the new image. Is there a way I can modify the plugin to delete the cache first so that it always gets fresh data?

Also, have you written or seen a plugin like imagebrowser that can support the Flash button?

**Comment by stzoid** [comment]
I came here to find the same answer about cache. Since no one answered your post, I figured it out. Here is the answer. In the config.js, where you set the imageBrowser_listURL, add the "math" function to generate a random value for the URL. 

config.imageBrowser_listUrl = "/images/images_list.json?"+Math.random();

**Comment by tareq-s** [comment]
@stzoid yup this works, but there must be a better way to do so, anyway I'm using your suggestion, thanks.

+1 for the issue.
